test/components/classifiers/test_zero_shot_document_classifier.py

```diff
@@ -12,19 +12,29 @@
 
 
 class TestTransformersZeroShotDocumentClassifier:
+    def test_init(self):
+        component = TransformersZeroShotDocumentClassifier(
+            model="cross-encoder/nli-deberta-v3-xsmall", labels=["positive", "negative"]
+        )
+        assert component.labels == ["positive", "negative"]
+        assert component.token == Secret.from_env_var(["HF_API_TOKEN", "HF_TOKEN"], strict=False)
+        assert component.multi_label is False
+        assert component.pipeline is None
+        assert component.classification_field is None
+
     def test_to_dict(self):
```

Comment:
Let's add another test that adds the component to a pipeline and performs the full serde roundtrip.

Reply:
@shadeMe, are there any references that I can use for this? I am not fully sure what this test should consist of

Reply:
Similar to [this](https://github.com/deepset-ai/haystack-experimental/blob/1787638e1b11bad26d2806f313b3812cfb6279f1/test/components/splitters/test_hierarchical_doc_splitter.py#L158) but instead of calling `to_dict` and `from_dict`, you'd call `dumps` and `loads`.

Reply:
I've added the test, let me know if that's what you were looking for :)